ui/components/prsidebar/prsidebar.go

```diff
@@ -71,6 +73,25 @@ func (m Model) Update(msg tea.Msg) (Model, tea.Cmd) {
 				return m, nil
 			}
 
+			m.inputBox, taCmd = m.inputBox.Update(msg)
+			cmds = append(cmds, cmd, taCmd)
+		} else if m.isApproving {
+			switch msg.Type {
+
+			case tea.KeyCtrlD:
+				if len(strings.Trim(m.inputBox.Value(), " ")) != 0 {
+					cmd = m.approve(m.inputBox.Value())
+				}
+				m.inputBox.Blur()
+				m.isApproving = false
+				return m, cmd
```

Comment:
This seems to require a comment to be entered, or else the approval won't occur. Yet the code here:
https://github.com/merkata/gh-dash/blob/d9033f7855730e536f5a3ecfbd8cb8e9f4c9268b/ui/components/prsidebar/approve.go#L33-L35
seems to indicate that approving with no comment is supposed to be supported. Do we want to support approving without a comment (like this PR itself was approved: https://github.com/dlvhdr/gh-dash/pull/399#pullrequestreview-2149546111)?

Reply:
I've opened a PR to enable approving without a comment:
- https://github.com/dlvhdr/gh-dash/pull/420